crates/solver/src/driver.rs

```diff
@@ -332,54 +339,17 @@ impl Driver {
             auction: competition_auction,
             solutions: rated_settlements
```

Comment:
unrelated to this PR, but I wish this was stored in descending order. This way `competition.solutions[0]` is **always** the winning solution. Maybe a small tweak we would do as a follow up (although this is a _breaking_ change that we would have to communicate - cc @harisang).